tests/testasciidoc.conf

```
@@ -861,3 +861,9 @@ data/lang-en-test.txt
 {'footer-style':'revdate'}
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+Generated chapter IDs normalized to ASCII test
```

```suggestion
!Generated chapter IDs normalized to ASCII test
```

Thanks for making this testcase and have verified it on my local machine. This change will disable it for now while I make modification to the syntax here to support optional python dependencies.